### PR TITLE
{Test} Live test pipeline update

### PR DESCRIPTION
**Pull request by qwordy**
**Description**

* Update index.html and update email content.
* Clean policy automatically.
* Fix "No log output captured" problem.

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
@Juliehzl @kairu-ms @haroldrandom For awareness.

File: scripts/live_test/CLITest.yml [hunk 1]
```diff
@@ -375,17 +375,20 @@ jobs:
         az login -u azureclitest@azuresdkteam.onmicrosoft.com -p $(AZURECLITESTPASSWORD)
         az account set -s 0b1f6471-1bf0-4dda-aec3-cb9272f09590
 
+        # Clean resources
+        python /home/vsts/work/1/s/scripts/live_test/clean_policy.py
```
Comment (fengzhou-msft) [inline scripts/live_test/CLITest.yml:379]: Can you use relative location?

**Review by fengzhou-msft — APPROVED** [review]
approve with minor comment

File: scripts/live_test/CLITest.yml (hunk 1 above)
Comment (qwordy) [reply scripts/live_test/CLITest.yml:379]: Yes. But we must pay attention to what current pwd is.

**Pull request merged**